src/lib/database/store.ts

```diff
@@ -1186,5 +1188,41 @@ const initializer = immer<Database>((set, get) => ({
     )
   },
 
+  addAcsSystem({
+    external_type,
+    name,
+    workspace_id,
+    created_at,
+    connected_account_ids,
+  }) {
+    const SYSTEM_TYPE_TO_DISPLAY_NAME: Record<AcsSystemExternalType, string> = {
+      pti_site: "PTI site",
+      alta_org: "Alta org",
+      brivo_account: "Brivo account",
+      salto_site: "Salto site",
+      hid_credential_manager_organization: "HID org",
+      visionline_system: "Visionline",
+      assa_abloy_credential_service: "Assa Abloy Credential Service",
+    }
```

Comment:
Any good reason that this is in the function? Could be in some `constants.ts` file somewhere

Reply:
https://github.com/seamapi/fake-seam-connect/pull/179/commits/4e515bf0dc019293ceda3f24c62da1e015d55f3f